### ISSUE TITLE
Restrict sparkle to auto-update the app if any paticular app is open 

### ISSUE THREAD

**Issue by ghost** [sparkle-project/Sparkle#1241]
Is there any way to restrict our application to update if any particular application is open.
Eg:- Restrict user to update an app if "Microsoft word" is open.

**Comment by kornelski** [comment]
Can you rephrase the question? I'm not sure what do you want to achieve.

**Comment by AmitPaloGlobant** [comment]
My application uses Microsoft Word to edit the .doc files. While editing the doc files if somebody updates my application meantime, then the changes done in the doc file is not saved. So, my goal is to restrict the user to update the application if Microsoft word is open. Just, let the user that need to save the changes first before updating the application.

**Comment by kornelski** [comment]
There is a delegate callback called before an update, and you can delay the update or ask user to save.
